lib/CloudinaryPlugin.js

```diff
@@ -136,12 +131,10 @@ class CloudinaryPlugin {
    * @see https://cloudinary.com/documentation/search_api
    */
   async search(request) {
```

Comment:
Node.js 6 does not support async/await

Reply:
I know but as you can see in the [index.js file](https://github.com/kuzzleio/kuzzle-plugin-cloudinary/blob/unit-testing/lib/index.js) first node-8 file is loaded and then, if an error is thrown, the node-6 compatible file is loaded. This file is built by babel after the install (post install script in [package.json file](https://github.com/kuzzleio/kuzzle-plugin-cloudinary/blob/unit-testing/package.json) )

Reply:
I missed that part. Nice! :+1: